Mappers/DbMapper.php

```diff
@@ -105,7 +105,7 @@ public function load(SearchCriteria $filter) : PaginationCollection
      */
     public function create(array $objectData)
     {
-        return $this->createObject($objectData);
+        return $this->createObject($this->buildObject($objectData)->toArray());
```

Comment:
You should get ArraySerializible into createObject method, and do to array there